riscv/src/compiler.rs

```diff
@@ -518,6 +518,17 @@ pil{
     { Y_b6 } in { bytes };
 }
 
+pil { col witness remainder; }
+
+// implements X = Y / Z, stores remainder in `remainder`.
```

Comment:
Oh we should also handle division by zero properly.

Reply:
What is division by zero in riscv?

Reply:
In our definition of division, it seems impossible to have a result if the divisor is 0. May not be the riscv mandated result, but it kinda covers it already.

Reply:
BTW, in riscv, division by zero sets all bits to 1.